### PR TITLE
python3Packages.python-louvain: fix test karate

### PR DESCRIPTION
**Pull request by erictapen**
Also add pandas, scipy to checkInputs

ZHF: #172160

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/python-louvain/default.nix [hunk 1]
```diff
@@ -13,10 +16,19 @@ buildPythonPackage rec {
     sha256 = "sha256-t7ot9QAv0o0+54mklTK6rRH+ZI5PIRfPB5jnUgodpWs=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/taynaud/python-louvain/pull/95/commits/c95d767e72f580cb15319fe08d72d87c9976640b.patch";
```
Comment (risicle) [inline pkgs/development/python-modules/python-louvain/default.nix:21]: Suggest giving this patch a `name = `

Comment (erictapen) [reply pkgs/development/python-modules/python-louvain/default.nix:21]: How would you name that patch? `karate.patch`?

Comment (SuperSandro2000) [reply pkgs/development/python-modules/python-louvain/default.nix:21]:
```suggestion
      # fix karake tests
      url = "https://github.com/taynaud/python-louvain/pull/95/commits/c95d767e72f580cb15319fe08d72d87c9976640b.patch";
```

Comment (erictapen) [reply pkgs/development/python-modules/python-louvain/default.nix:22]: done!

Comment (risicle) [reply pkgs/development/python-modules/python-louvain/default.nix:22]: What is does - `fix-karate-test-networkx-2.7.patch`

Comment (erictapen) [reply pkgs/development/python-modules/python-louvain/default.nix:23]: Added that :+1:

**Comment by SuperSandro2000** [comment]
@ofborg build python3Packages.python-louvain

**Pull request merged**